### DISCUSSION TITLE
Need help with save function

### DISCUSSION THREAD

**Discussion by CXXXV** [tinymce/tinymce#9125] [General]
I have a request from user to add a 'SAVE' function. I added the plugin and the toolbar. When selecting SAVE I get the 'no form element found' error.

After searching for quite a while all I can find is the issue with submit conflict. I am not submitting any form in my case. All I need is for the save button to behave like a standard save dialog where the user chooses local save - don't want to use database or server store. The file needs to also have any formatting associated with it. Similar to autosave but with a location specified.

And then of course we will need a way to load.

**Comment by TheSpyder** [comment]
The save plugin is designed to submit HTML forms.
https://www.tiny.cloud/docs/tinymce/6/save/

However if you configure callback functions those will be used instead of the default behaviour. That makes the feature completely customisable.
https://github.com/tinymce/tinymce/blob/236891ffcdf2fd1b698d020e67c31838fb110019/modules/tinymce/src/plugins/save/main/ts/core/Actions.ts#L27-L31
